ios/MeasureSDK/MeasureSDK/Config/Config.swift

```diff
@@ -33,5 +35,7 @@ struct Config: InternalConfig, MeasureConfig {
         self.eventsBatchingIntervalMs = 30000 // 30 seconds
         self.maxEventsInBatch = 500
         self.sessionEndThresholdMs = 60000 // 60 seconds
+        self.longPressTimeout = 0.5 // 0.5 seconds
```

Comment:
this is 500 ms as per [flutter documentation](https://api.flutter.dev/flutter/gestures/LongPressGestureRecognizer-class.html#:~:text=Recognizes%20when%20the%20user%20has,for%20500ms%20until%20it's%20recognized.).